farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.html

```diff
@@ -130,9 +134,25 @@ <h1 class="text-center">Seeding Report</h1>
                     this.reportVisible = true
                     let link = '/log.json?type=farm_seeding&timestamp[ge]=' + dayjs(this.startDate).unix() + '&timestamp[le]=' + dayjs(this.endDate).unix()
 
-                    getAllPages(link, this.seedingLogs).then(() => {
-                        this.stillLoading = false
-                    })
+                    getAllPages(link, this.seedingLogs)
+                        .then(() => {
+                            this.stillLoading = false
+                        })
+                        .catch((err) => { 
+                                if (err.response) {
```

Comment:
These conditionals can be simplified.  It looks like you are doing the same thing in the catch, no matter what happens.  So you can eliminate the if/else if/else in the catch entirely and just set `this.errBanner=true`.

Reply:
This should be removed now.